experimental/packages/opentelemetry-sdk-metrics-base/src/view/Aggregation.ts

```diff
@@ -24,19 +28,91 @@ import { InstrumentDescriptor } from '../InstrumentDescriptor';
  * Aggregation provides a set of built-in aggregations via static methods.
  */
 export abstract class Aggregation {
-  // TODO: define the actual aggregator classes
-  abstract createAggregator(instrument: InstrumentDescriptor): unknown;
+  abstract createAggregator(instrument: InstrumentDescriptor): Aggregator<Maybe<Accumulation>>;
 
-  static None(): Aggregation {
-    return NONE_AGGREGATION;
+  static Drop(): Aggregation {
+    return DROP_AGGREGATION;
+  }
+
+  static Sum(): Aggregation {
+    return SUM_AGGREGATION;
+  }
+
+  static LastValue(): Aggregation {
+    return LAST_VALUE_AGGREGATION;
+  }
+
+  static Histogram(): Aggregation {
+    return HISTOGRAM_AGGREGATION;
+  }
+
+  static Default(): Aggregation {
+    return DEFAULT_AGGREGATION;
   }
 }
 
-export class NoneAggregation extends Aggregation {
+export class DropAggregation extends Aggregation {
+  static kDefault = new DropAggregator();
```

Comment:
What is the meaning of the k in the name name `kDefault`?

Reply:
It means "constant" so we don't have to CAPITALIZE all the characters in the name. It's should not be the public API I'd be fine with both patterns (prefixing with `k`, or capitalizing the name).

Reply:
I prefer capital since that's what we've done other places in the code.

Reply:
yeah, updated with capitalized names.